java/core/core.iml

```diff
@@ -13,7 +13,6 @@
     </content>
     <orderEntry type="inheritedJdk" />
     <orderEntry type="sourceFolder" forTests="false" />
-    <orderEntry type="module" module-name="gateway-core" />
```

Comment:
Ah, that has been removed too in https://github.com/Worklytics/psoxy/pull/138/files#diff-311e2f8f3a14f67cf4b6bb78c8f7133e8a9ddf832fd907c0857f8d478f584733